git/week2/main.js

```diff
@@ -0,0 +1,49 @@
+
+//console.log('Script loaded');
+
+const products = getAvailableProducts();
+const productsUl = document.querySelector('section.products ul');
+//console.log(productsUl);
+
+let e=document.querySelector(".country>select")
```

Comment:
this is supposed to be reacting to searches not the country select field changing :)